addons/website/views/snippets/s_color_blocks_2.xml

```diff
@@ -5,15 +5,13 @@
     <section class="s_color_blocks_2">
         <div class="container-fluid">
             <div class="row">
-                <div class="col-lg-6 o_cc o_cc3 text-center">
-                    <i class="fa fa-shield fa-5x m-3"/>
-                    <h2>A color block</h2>
+                <div class="col-lg-6 o_cc o_cc4">
+                    <h2 class="h3-fs">A color block</h2>
```

Comment:
Just a question: 
Didn't you forget to add the `h3-fs` class in the new page template "Big Boxes" (so in the `new_page_template_templates.xml` file? :slightly_smiling_face: 

```xml
!-- s_color_blocks_2 -->

<template id="new_page_template_s_color_blocks_2" inherit_id="website.s_color_blocks_2" primary="True"/>

<template id="new_page_template_landing_s_color_blocks_2" inherit_id="website.new_page_template_s_color_blocks_2" primary="True">
    <xpath expr="//h2" position="replace">
        <h2>Crystal Clear Sound</h2>
    </xpath>
</template>
```
The first title is bigger than the second :wink: 
![image](https://github.com/user-attachments/assets/eb16fcee-7fc3-4373-b0e9-ed4895a539eb)

Reply:
Good catch ! 👍 
I've updated and rebased both branches, thanks !